stack

```diff
@@ -24,6 +24,10 @@ export GETH_DOCKER_IMAGE=${GETH_DOCKER_IMAGE:="ethereum/client-go:v1.13.5"}
 export FAUCET_DOCKER_IMAGE=${FAUCET_DOCKER_IMAGE:="hive-faucet"}
 # what is the docker image that we use for solc compiler?
 export SOLC_DOCKER_IMAGE=${SOLC_DOCKER_IMAGE:="hive-solc"}
+# run everything local in dev mode
```

Comment:
maybe comment should be "if this variable is set, use a local geth instance rather than geth in docker. needed on macos"